README.md

```diff
@@ -593,10 +593,11 @@ public static Range between(final double bottom, final boolean openBottom, final
 ```
 
 Once you have constructed appropriate `Patterns` matchers with these methods, you can use the
-following form of `Machine.addRule()` to add them to your machine:
+following methods to add to or delete from your machine:
```

Comment:
Maybe add a line saying that the cautions expressed in the description of `deleteRule` apply here, i.e. the patterns in the `deletePatternRule` call aren't required to match those in the corresponding `add`.